tools/TwigRenderer.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+
+namespace OC_Blog\Tools;
+
+
+class HtmlResponse {
+	protected $twig;
+	protected $statusCode;
+	protected $data;
+	protected $twigPath;
+
+	public function __construct($statusCode, $twigPath, $data)
+	{
+		$this->statusCode = $statusCode;
+		$this->data = $data;
+		$this->twigPath = $twigPath;
+	}
+	public function setTwig($twig)
+	{
+		$this->twig = $twig;
+	}
+
+	public function addData($key, $value)
+	{
+		$this->data[$key] = $value;
+	}
+
+	public function render() {
+		echo $this->twig($this->twigPath, $this->data);
```

Comment:
Codacy found an issue: [All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$this'.](https://app.codacy.com/manual/PatOpen/OC_Blog/pullRequest?prid=6061935)